vector/src/main/java/im/vector/app/features/location/UrlMapProvider.kt

```diff
@@ -30,11 +29,12 @@ class UrlMapProvider @Inject constructor(
         private val session: Session,
         private val rawService: RawService
 ) {
-    private val keyParam = "?key=${BuildConfig.mapTilerKey}"
+    // Tchap: Disable MapTiler
+//    private val keyParam = "?key=${BuildConfig.mapTilerKey}"
```

Comment:
you can just pass an empty string instead of commenting this variable to restore the next changes